inc/Engine/Preload/Subscriber.php

```diff
@@ -447,6 +447,10 @@ public function unlock_url( string $url ) {
 	 * @return array|false
 	 */
 	public function add_preload_excluded_uri( $regexes ): array {
+		if ( (bool) ! $this->options->get( 'manual_preload', true ) ) {
+			return false;
```

Comment:
@CrochetFeve0251 If I believe the docblock, it returns false or an array. 
If you return the original value, like it was before, it would process the URL and add it to the database, which is what we want to avoid here.

Returning `false` means it won't be added to the database

Reply:
Umm I've changed the way to do it, bailing out before, reaching this function.